src/UXCapture.js

```diff
@@ -45,6 +47,8 @@ const UXCapture = {
 	create: config => {
 		_onMark = config.onMark || NOOP;
 		_onMeasure = config.onMeasure || NOOP;
+		_markSelector = config.markSelector;
+		_elementSelector = config.elementSelector;
```

Comment:
I'm curious why we would want to support global mark/element selectors - at first look, it seems like it complicates the behavior quite a bit - are these global selectors implicit 'expected' requirements for every view? I'd like to force the client to be as explicit as possible for every view to avoid unexpected behavior, which would also make the code cleaner by only dealing with `markSelector` and `elementSelector` closer to where marks are handled in `ExpectedMark`

Reply:
We don't have to support all 3 interfaces, I just wanted to add solution that works with all the examples we discussed.

In reality, I believe markSelector is just a hack and has no good reason to exist except for being easiest to add it to existing configs used with current version of UX Capture.

And elementSelector to be both global (e.g. established rule for entire site) and defined for individual element (more flexibility, less centralization of solution) is a good idea. Both can be implemented by React client.

Reply:
I think my main concern with an `elementSelector` global is that it means that each view instance isn't fully in control of defining its expected marks, which leads to ambiguous behavior, e.g. "Why isn't my 'primary content loaded' measure firing even though all of the marks I passed into the view configuration have fired?!?"

If we can always say "the measures for a particular view always correspond to the marks passed into `startView`", the tool will be easier to use and understand.

Reply:
Global selector function can rely on convention established by specific site.

For example in [element timing example](https://github.com/meetup/ux-capture/blob/bd294bb074f805b1b7117a29bda9091506706067/examples/selectors-elementtiming.html#L27) this convention is element's name set as `elementtiming` attribute's value on the element. Same goes for universal selector example which [uses class name](https://github.com/meetup/ux-capture/blob/bd294bb074f805b1b7117a29bda9091506706067/examples/selectors-universal.html#L27) as convention.

Because global selector takes element object from zone definition as an attribute it allows both convention and flexibility.

Reply:
BTW, in current implementation, individual elementSelectors override global one in case view wants to deviate from a convention.